lib/Perl/MinimumVersion.pm

```diff
@@ -784,6 +785,86 @@ sub _heredoc_indent {
 	});
 }
 
+# Postfix dereference new (and experimental) in 5.20.
+sub _postfix_deref {
+	shift->Document->find_first( sub {
+		$DB::single = 1;
+		my $main_element=$_[1];
+		$main_element->isa('PPI::Token::Symbol') or return '';
+		(_get_resulting_sigil($main_element) || '') eq '$'
+			or return '';
+		my $operator = $main_element->snext_sibling()
+			or return '';
+		return '' unless
+			$operator->isa( 'PPI::Token::Operator' ) &&
+			$operator->content() eq '->';
+		my $next = $operator->snext_sibling()
+			or return '';
+		my $content = $next->content();
+		# FIXME: The below Byzantine mess could be considerably
+		# simplified if we could rely on a PPI that supported
+		# postfix dereferencing. My read of PPI's Changes file
+		# says this is 1.238, which is considerably ahead of
+		# Perl-MinimumVersion's requirement of PPI 1.215. -- trwyant
```

Comment:
@Grinnz, @neilb, thoughts on bumping the minimum PPI version to 1.238?

Reply:
Seems quite reasonable to me